libpod/util.go

```diff
@@ -142,29 +142,35 @@ func JSONDeepCopy(from, to interface{}) error {
 	return json.Unmarshal(tmp, to)
 }
 
-func dpkgVersion(path string) string {
+func queryPackageVersion(cmdArg ...string) string {
 	output := unknownPackage
-	cmd := exec.Command("/usr/bin/dpkg", "-S", path)
+	cmd := exec.Command(cmdArg[0], cmdArg[1:]...)
```

Comment:
I know the code is all fine, but just to make sure in the future, you probably ought to add a len check for cmdArg before this line.

Reply:
Hey @TomSweeneyRedHat , thanks for pointing this out.

How do you think about this? I'm not sure if we should have a max len for `cmdArg` though.

```go
func queryPackageVersion(cmdArg ...string) string {                             
    output := unknownPackage
    if 1 < len(cmdArg) {                                                    
        cmd := exec.Command(cmdArg[0], cmdArg[1:]...)                               
        if outp, err := cmd.Output(); err == nil {                                  
            output = string(outp)                                                   
        }                           
    }                                                
    return strings.Trim(output, "\n")                                           
}
```

Reply:
LGTM, no max check.

Reply:
@rhatdan, awesome, I've amended my PR. Please let me know if you have any other suggestions. :+1: